presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -26,6 +26,10 @@ singleExpression
     : expression EOF
     ;
 
+standalonePathSpecification
```

Comment:
We should name this `single` to be consistent with `singleExpression` and `singleStatement` (or rename the others if `standalone` is a better name)

Reply:
For statements, single makes sense because we could have a version that parses multiple statements (although, we might call that one "block" or something like that).

For expressions, yes, standalone is a more accurate name. There's no such a thing as multiple expressions. Similarly for pathSpecification.